### PR TITLE
frontend: Show messages on executing operation

### LINKED ISSUE #32: bank: handle corner cases for deposit and withdraw
Corner cases for deposit and withdraw need to be handled like: 

Balance: 20, Withdraw 15 -> Shows message: "Operation completed successfully."
Balance: 10, Deposit 13 -> Shows message: "Operation completed successfully."
Balance: 20, Withdraw 23 -> Shows error: "Insufficient  amount in your account."

### PR DESCRIPTION
**Pull request by martinmilev**
Corner cases for deposit and withdraw handled.

> Fixes #32

screenshot from 2017-01-24 16-47-10

screenshot from 2017-01-24 16-47-21

### CODE REVIEW

File: frontend/app/app.js [hunk 1]
```diff
@@ -1,43 +1,52 @@
...
-    .when('/', {
-      templateUrl: 'view/index_page.view.html'
-    })
-    .when('/v1/transaction', {
-      templateUrl: 'view/transaction_history_page.view.html'
-    })
-    .otherwise({
-      redirectTo: '/'
-    });
+          .when('/', {
```
Comment (mgenov) [inline frontend/app/app.js]: Formatting should be kept on 2 spaces.

Comment (mgenov) [inline frontend/app/app.js]: Formatting should be kept on 2 spaces.

File: frontend/app/app.js [hunk 2]
```diff
@@ -1,43 +1,52 @@
...
-app.controller('TransactionsHistoryCtrl', function($scope, $http) {
+app.controller('TransactionsHistoryCtrl', function ($scope, $http) {
   // an empty list of transactions should be displayed 
   // before loading of the history
   $scope.transactions = [];
   $http.get("/v1/transactions")
-    .then(function(response) {
-      $scope.transactions = response.data;
-    });
+          .then(function (response) {
```
Comment (mgenov) [inline frontend/app/app.js]: .

File: frontend/app/app.js [hunk 3]
```diff
@@ -1,43 +1,52 @@
...
-app.controller('HomePageCtrl', function($scope, $http) {
+app.controller('HomePageCtrl', function ($scope, $http) {
   $scope.account = {};
   $scope.operation = {type: "deposit"};
 
   $http.get("/v1/useraccount")
-    .then(function(response) {
-      $scope.account = response.data;
-  });
+          .then(function (response) {
```
Comment (mgenov) [inline frontend/app/app.js]: .

**Comment by martinmilev** [comment]
Screenshots added.

**Pull request merged**